lib/streamlit/net_util.py

```diff
@@ -45,16 +45,17 @@ def get_external_ip():
 
     response = _make_blocking_http_get(_AWS_CHECK_IP, timeout=5)
 
-    if response is None:
+    if _looks_like_an_ip_adress(response):
+        _external_ip = response
+    else:
         LOGGER.warning(
             # fmt: off
             "Did not auto detect external IP.\n"
             "Please go to %s for debugging hints.",
             # fmt: on
             util.HELP_DOC
         )
-    else:
-        _external_ip = response.strip()
+        _external_ip = None
```

Comment:
Not sure if we need this line since `_external_ip` is initialized to `None` outside of the function and if it ever has a value we won't get here.

Reply:
I  know. I just thought this was more readable. It makes it absolutely clear what the value `_external_ip` will be. You don't have to read the whole function to understand it.